### DISCUSSION TITLE
How to send message to default exchange?

### DISCUSSION THREAD

**Discussion by Gauthamraju31** [rabbitmq/amqp091-go#91] [Q&A] [answered]
I am trying to send a message to the default exchange by setting the exchange name as "" (as mentioned by amqp) which is considered the AMQP Default Exchange. However I am greeted with an error as following - 

`  [ERROR] Exchange Declare: Exception (403) Reason: "ACCESS_REFUSED - operation not permitted on the default exchange"
`  
   I have tried to call Declare exchange passive function and yet no luck. Could someone explain me how do I achieve it with appropriate properties?

**Comment by Gsantomaggio** [comment, marked as answer]
```golang
channel.Publish(
			"",   // default Exchange 
			routingKey, // queue name
			false,      // mandatory
			false,      // immediate
			amqp.Publishing{
				Headers:         amqp.Table{},
				ContentType:     "text/plain",
				ContentEncoding: "",
				Body:            []byte(body),
				DeliveryMode:    amqp.Transient, // 1=non-persistent, 2=persistent
				Priority:        0,              // 0-9
				// a bunch of application/implementation-specific fields
			},
```
